manifests/provider/ad.pp

```diff
@@ -124,12 +124,13 @@
   Optional[String]                                           $ldap_idmap_default_domain_sid            = undef,
   Optional[String]                                           $ldap_idmap_default_domain                = undef,
   Optional[Boolean]                                          $ldap_idmap_autorid_compat                = undef,
-  Optional[Integer[1]]                                       $ldap_idmap_helper_table_size             = undef
+  Optional[Integer[1]]                                       $ldap_idmap_helper_table_size             = undef,
+  Stdlib::Absolutepath                                       $conf_file_path                           = simplib::lookup('sssd::conf_file_path', { 'default_value' =>  '/etc/sssd/sssd.conf' })
```

Comment:
This won't work without an inherits (which can't be done) so I guess needs to default to Optional and then select it after the `include` below.